prime-router/src/main/kotlin/azure/BatchFunction.kt

```diff
@@ -38,10 +44,12 @@ class BatchFunction {
             val maxBatchSize = receiver.timing?.maxReportCount ?: defaultBatchSize
             val actionHistory = ActionHistory(event.eventAction.toTaskAction(), context)
             actionHistory.trackActionParams(message)
-            context.logger.info(
-                "BatchFunction for $message" +
-                    " will look for tasks created after $backstopTime, for receiver ${event.receiverName}"
+            backstopTime = OffsetDateTime.now().minusMinutes(
+                WorkflowEngine.getBatchLookbackMins(
```

Comment:
I separated out the calculation of the number of "lookback minutes" because its much easier to write complete unit tests for it, than a function that returns an exact time.